auth/src/main/java/com/firebase/ui/auth/ui/idp/CredentialSignInHandler.java

```diff
@@ -77,15 +78,19 @@ public void onFailure(@NonNull Exception e) {
             } else {
                 mActivityHelper.dismissDialog();
                 Log.e(TAG, "Unexpected exception when signing in with credential",
-                        task.getException());
+                      task.getException());
             }
         } else {
             mActivityHelper.dismissDialog();
 
             FirebaseUser firebaseUser = task.getResult().getUser();
-            SmartlockUtil.saveCredentialOrFinish(mActivity, mSaveCredentialsResultCode,
-                    mActivityHelper.getFlowParams(), firebaseUser,
-                    null /* password */, mResponse.getProviderType());
+            SmartlockUtil.saveCredentialOrFinish(mActivity,
+                                                 mSaveCredentialsResultCode,
+                                                 new Intent(),
```

Comment:
nit: I'd prefer to pass `null /* intent */` here and have `saveCredentialOrFinish` handle the `null` case.